<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RuleEditor</class>
 <widget class="QDialog" name="RuleEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1256</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4" colspan="2">
    <widget class="QLineEdit" name="edt_rule_name"/>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Rules:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QPushButton" name="btn_rules_add">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btn_rules_remove">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QListWidget" name="lst_rules"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="4">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Target</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QToolBox" name="toolbox">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>946</width>
            <height>234</height>
           </rect>
          </property>
          <attribute name="label">
           <string>(DEF/F0) Designator</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QPlainTextEdit" name="txt_fields_designator"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>96</width>
            <height>86</height>
           </rect>
          </property>
          <attribute name="label">
           <string>(Def/F1) Name/Value</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <widget class="QPlainTextEdit" name="txt_fields_name"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>96</width>
            <height>86</height>
           </rect>
          </property>
          <attribute name="label">
           <string>(F2) Footprint</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <widget class="QPlainTextEdit" name="txt_fields_footprint"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>96</width>
            <height>86</height>
           </rect>
          </property>
          <attribute name="label">
           <string>(F3) Datasheet</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="0" column="0">
            <widget class="QPlainTextEdit" name="txt_fields_datasheet"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_5">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>96</width>
            <height>86</height>
           </rect>
          </property>
          <attribute name="label">
           <string>(F4) ID</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_12">
           <item row="0" column="0">
            <widget class="QPlainTextEdit" name="txt_fields_id"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_6">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>96</width>
            <height>86</height>
           </rect>
          </property>
          <attribute name="label">
           <string>(F5) MPN</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_13">
           <item row="0" column="0">
            <widget class="QPlainTextEdit" name="txt_fields_mpn"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_7">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>96</width>
            <height>86</height>
           </rect>
          </property>
          <attribute name="label">
           <string>(F6) Manufacturer</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_14">
           <item row="0" column="0">
            <widget class="QPlainTextEdit" name="txt_fields_manufacturer"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_8">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>96</width>
            <height>86</height>
           </rect>
          </property>
          <attribute name="label">
           <string>(F7) Display Value</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_15">
           <item row="0" column="0">
            <widget class="QPlainTextEdit" name="txt_fields_display_value"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_9">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>96</width>
            <height>86</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Description</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QPlainTextEdit" name="txt_fields_description"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_16">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>96</width>
            <height>86</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Target Library Name</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_20">
           <item row="0" column="0">
            <widget class="QPlainTextEdit" name="txt_fields_library_name"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Category</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Proposed Categories:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btn_category_add">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btn_category_remove">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>linked categories:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QListWidget" name="lst_category_proposed"/>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QListWidget" name="lst_category_used"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Source Devices</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="2" column="0">
        <widget class="QPushButton" name="btn_source_dev_add">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btn_source_dev_remove">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QListWidget" name="lst_source_dev_proposed"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Source device preference:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Proposed Source Device preference:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QListWidget" name="lst_source_dev_used"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Keywords</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QToolBox" name="toolBox_2">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_13">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>946</width>
            <height>528</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Octopart Description</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_17">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Priority Up:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QPlainTextEdit" name="txt_key_desc_up"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Priority Down:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QPlainTextEdit" name="txt_key_desc_down"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_14">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>159</width>
            <height>234</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Octopart Specification</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_19">
           <item row="4" column="0">
            <widget class="QPlainTextEdit" name="txt_key_spec_down"/>
           </item>
           <item row="2" column="0">
            <widget class="QPlainTextEdit" name="txt_key_spec_up"/>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Priority Up:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Priority Down:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_15">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>159</width>
            <height>234</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Octopart Footprint</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_18">
           <item row="4" column="0" colspan="2">
            <widget class="QPlainTextEdit" name="txt_key_foot_down"/>
           </item>
           <item row="1" column="0" rowspan="2" colspan="2">
            <widget class="QPlainTextEdit" name="txt_key_foot_up"/>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Priority Up:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Priority Down:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RuleEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RuleEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
